optuna/study/study.py

```diff
@@ -651,13 +654,26 @@ def df(x):
 
         if values is not None:
             values, values_conversion_failure_message = _check_and_convert_to_values(
```

Comment:
Not a strong opinion and can be a separate PR but we can improve locality by moving the definition of `_check_and_convert_to_values` to this file now.

Reply:
It makes sense. 137476c does it.

Reply:
Done in 137476c. :+1: